sample_inventories/ccloud.yml

```diff
@@ -0,0 +1,153 @@
+---
+### Confluent Cloud Connection
+##
+## The following is an example inventory file of the configuration required for setting up Confluent Platform with:
+# all components connecting to Kafka and Schema Registry in Confluent Cloud
+
+all:
+  vars:
+    ansible_connection: ssh
+    ansible_user: ec2-user
+    ansible_become: true
+    ansible_ssh_private_key_file: /home/ec2-user/guest.pem
+
```

Comment:
```suggestion
    # Make sure ssl_enabled and sasl_protocol variables are set as below so that no incorrect security properties get added
    ssl_enabled: false
    sasl_protocol: none
    
    # Can still set component level variables like
    kafka_connect_ssl_enabled: true
    kafka_rest_ssl_enabled: true
    ksql_ssl_enabled: true
    control_center_ssl_enabled: true
 
```